src/utl/src/Utl.tcl

```diff
@@ -0,0 +1,99 @@
+global MAN_PATH 
+set MAN_PATH ""
+
+proc get_input { } {
+  # Get the relative path from the user
+  puts "Please enter an optional relative path to the cat folders:"
+  flush stdout
+  gets stdin relative_path
+
+  # Convert the relative path to an absolute path
+  set absolute_path [file join [pwd] $relative_path]
+
+  # Display the absolute path
+  puts "Absolute Path: $absolute_path"
+  if { [check_valid_path $absolute_path] } {
+    return $absolute_path
+  }
+  return ""
+}
+
+proc check_valid_path { path } {
+  if {[file isdirectory $path]} {
+    return true 
+  } else {
+    puts "Invalid path, please retry."
+    return false
+  }
+}
+
+proc check_valid_man_path { path } {
+  if {[file isdirectory "$path/man1"]} {
+    return true
+  } else {
+    puts "Invalid man path, please retry."
+    return false
+  }
+}
```

Comment:
these are helper function and should not be exposed to the global namespace.

Reply:
@gadfort I made some changes, does it look okay?